include/openssl/ssl3.h

```diff
@@ -307,9 +307,6 @@ extern "C" {
 /* Set if extended master secret extension required on renegotiation */
 # define TLS1_FLAGS_REQUIRED_EXTMS               0x1000
 
-/* Set if this is the QUIC handshake layer */
-# define TLS1_FLAGS_QUIC                         0x2000
```

Comment:
Given that this is the location where all the rest of the flags are defined, I think at least a comment here is worthwhile saying that 0x2000 is reserved for the internal flag `TLS1_FLAGS_QUIC`. Otherwise I think there is a risk that if we add a new flag in the future we will inadvertently reuse the same flag value.

Reply:
Good point.